packages/examples/src/demo/blocks/input.tsx

```diff
@@ -56,6 +66,32 @@ export const useInput = (
     });
   };
 
+  const importFromJsonLogic = () => {
+    try {
+      // Parse the JSON string with proper typing
+      const jsonLogicObj: unknown = JSON.parse(jsonLogicStr);
+      
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+      const tree = Utils.loadFromJsonLogic(jsonLogicObj as JsonLogicFunction, state.config);
```

Comment:
The correct type is `JsonLogicTree`
https://github.com/ukrbublik/react-awesome-query-builder/blob/fdc33cabc500179ae58b4a0e35143fcc3c153ac1/packages/core/modules/index.d.ts#L541
```suggestion
      const [tree, logicErrors] = Utils._loadFromJsonLogic(jsonLogicObj as JsonLogicTree, state.config);
      if (logicErrors?.length) {
        throw logicErrors; // TODO: use in `setJsonLogicErrors`
      }
```